File: 1-TwoSum/1-TwoSum.py
# Last updated: 7/14/2026, 2:17:21 PM
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d={}
        for i in range(0,len(nums)):
            value=nums[i]
            difference = target-value
            if value not in d:
                d[difference]=i
            else:
                current_index = i
                previous_index =d[value]
                return [current_index,previous_index]

File: 1-TwoSum/1-TwoSum.java
// Last updated: 7/14/2026, 2:17:24 PM
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] arr = new int[2];
        for (int i = 0; i < nums.length;i++){
            for (int j = 1; j < nums.length;j++) {
                if(i!=j){
                if ((nums[i] + nums[j]) == target) {
                    arr[0] = i;
                    arr[1] = j;
                }
                }
            }
        }
        return arr;
    }
}